rewrite-maven/src/main/java/org/openrewrite/maven/RemoveDuplicateDependencies.java

```diff
@@ -125,6 +125,9 @@ private boolean isManagedDependenciesTag() {
             }
 
             private @Nullable DependencyKey getManagedDependencyKey(Xml.Tag tag) {
+                if (tag.getChildValue("scope").map("import"::equalsIgnoreCase).orElse(false)) {
+                    return DependencyKey.from(tag);
+                }
                 ResolvedManagedDependency resolvedDependency = findManagedDependency(tag);
```

Comment:
`findManagedDependency()` does not actually find imported boms. It seems they are not included in dependency management by the resolver.